src/RealtimeEffectPanel.cpp

```diff
@@ -458,6 +469,25 @@ namespace
          effectStateUI.Toggle( *mProject );
       }
 
+      void OnEnableButtonClicked(wxCommandEvent& event)
+      {
+         auto pButton = static_cast<ThemedButtonWrapper<wxBitmapButton>*>(mEnableButton);
+         auto index = pButton->GetBitmapIndex();
+         bool wasEnabled = (index == bmpEffectOn);
+
+         if (mSettingsAccess) {
```

Comment:
I think this should become a `RealtimeEffectState::SetActive`. That specific method should also Publish the message

Reply:
Done.